install.sh

```diff
@@ -144,6 +144,18 @@ Install()
     # If update, start Wazuh
     if [ "X${update_only}" = "Xyes" ]; then
         WazuhUpgrade $INSTYPE
```

Comment:
Question: did you consider adding this logic to this `WazuhUpgrade` function (it's on src/init/wazuh/wazuh.sh)? It seems to have other related tasks.

Reply:
no, I didn't consider it, but I could move the logic before the end of the `WazuhUpgrade` function.

Reply:
done